### ISSUE TITLE
Why is change_plan_swin deleted?

### ISSUE THREAD

**Issue by JasonRichard** [282857341/nnFormer#82]
Thanks for your extraoridinary work!

It took me some time to figure out the train-test split in the paper and your code correspondingly.
Until I saw the following issue:
[Why all test samples appear in training dataset for Synapse?](https://github.com/282857341/nnFormer/issues/4)

However the file was somehow deleted in 9 Oct 2021, the following version committed on 2 Sep 2021 seems right.
[Update change_plan_swin.py](https://github.com/282857341/nnFormer/commit/f5170ed2d37ba13e80579386c9d3d49273c33ed1)

Did you deleted it accidently by commit?

best,

**Comment by 282857341** [comment]
I'm sorry for the inconvenience the manner brings to you.  The change_plan_swin.py is not used now, because I think modifying the plan files is cumbersome and you need to run the change_plan_swin.py each time you modify the parameter.

Now, we change the division of the training set and validation set directly in the trainer file as the code below.
https://github.com/282857341/nnFormer/blob/d48aaeb05612a09538f70c38a2357149ea3a7ac0/nnformer/training/network_training/nnFormerTrainerV2_nnformer_synapse.py#L338-L340

And we change the batch size and patch size in the code below:
https://github.com/282857341/nnFormer/blob/d48aaeb05612a09538f70c38a2357149ea3a7ac0/nnformer/run/default_configuration.py#L61-L67

**Comment by JasonRichard** [comment]
Sorry I am still confused. 
It looks like the splits will not change varing to fold (when fold is smaller than 5)?
Because you alter it after loading the pickle file. If `nnUNet_plan_and_preprocess` is run, the `len(splits)` would be 5 and the following branch would always be true.
https://github.com/282857341/nnFormer/blob/d48aaeb05612a09538f70c38a2357149ea3a7ac0/nnformer/training/network_training/nnFormerTrainerV2_nnformer_synapse.py#L341-L345

Does it mean that we only need to run one fold in this circumstance?


The only branch splits varing to the fold is that 
`It exists a splits_final.pkl that contains less fold than the fold than we ask for.`
Which goes branch Line322
https://github.com/282857341/nnFormer/blob/d48aaeb05612a09538f70c38a2357149ea3a7ac0/nnformer/training/network_training/nnFormerTrainerV2_nnformer_synapse.py#L332-L335
and 
https://github.com/282857341/nnFormer/blob/d48aaeb05612a09538f70c38a2357149ea3a7ac0/nnformer/training/network_training/nnFormerTrainerV2_nnformer_synapse.py#L346-L358

**Comment by 282857341** [comment]
Yes, we only need to run one fold and we do this because we need to follow the split of other methods for each dataset. 

If you need to run the cross-validation, you can load the splits from the original plan file.
